pkg/controller/datavolume-controller.go

```diff
@@ -797,48 +649,45 @@ func (c *DataVolumeController) getPodFromPvc(namespace string, pvcUID types.UID)
 	return nil, errors.Errorf("Unable to find pod owned by UID: %s, in namespace: %s", string(pvcUID), namespace)
 }
 
-func (c *DataVolumeController) updateProgressUsingPod(dataVolumeCopy *cdiv1.DataVolume, pod *corev1.Pod) {
+func updateProgressUsingPod(dataVolumeCopy *cdiv1.DataVolume, pod *corev1.Pod) error {
 	httpClient := buildHTTPClient()
 	// Example value: import_progress{ownerUID="b856691e-1038-11e9-a5ab-525500d15501"} 13.45
 	var importRegExp = regexp.MustCompile("progress\\{ownerUID\\=\"" + string(dataVolumeCopy.UID) + "\"\\} (\\d{1,3}\\.?\\d*)")
 
-	port, err := c.getPodMetricsPort(pod)
+	port, err := getPodMetricsPort(pod)
 	if err == nil {
 		url := fmt.Sprintf("https://%s:%d/metrics", pod.Status.PodIP, port)
```

Comment:
while we're here:  maybe we should not attempt to get status if `pod.Status.PodIP == ""`.  This spams the logs and confuses QA.

Reply:
Good point, I will add that check to only attempt if we have a pod ip.